services/udf-python/src/io/aklivity/zillabase/flask_routes.py

```diff
@@ -0,0 +1,61 @@
+from flask import Flask, jsonify
+
+app = Flask(__name__)
+
+functions = []
+
+@app.route("/python/methods", methods=["GET"])
```

Comment:
`methods` or `functions` for  python `udf` server?

Reply:
actually I think both should be functions since from risingwave point of view it is functions.